framework/helpers/Url.php

```diff
@@ -15,4 +15,31 @@
  */
 class Url extends BaseUrl
 {
+    /**
+     * @param        $params  Array of values to convert into http query
+     * @param string $keyfather Inheritance of keys
+     * @param string $response
+     * @param bool   $ltrim
+     *
+     * @return string
+     */
+    public static function custom_http_build_query($params, $keyfather = '', $response = '', $ltrim = true) {
```

Comment:
Did you wrote this yourself or is there a source to referenced? Could you describe the arguments a bit more? It's not clear how they're used. Shouldn't they params be compatible with built-in `http_build_query`?